modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -210,6 +214,30 @@ public static boolean containsSqlScriptDelimiters(String script, String delim) {
 		return false;
 	}
 
+	/**
+	 * Load script from classpath and apply it to the given database
+	 *
+	 * @param databaseDelegate database delegate for script execution
+	 * @param initScriptPath   the resource to load the init script from
+	 */
+	public static void runInitScript(DatabaseDelegate databaseDelegate, String initScriptPath) {
+		try {
+			URL resource = Thread.currentThread().getContextClassLoader().getResource(initScriptPath);
```

Comment:
Why do you need to use `Thread` here and can't use `ScriptUtils.class.getResource("foo")`?

Reply:
It's an old code that previously was in ContainerDatabaseDriver.runInitScriptIfRequired(...). It's because module 'jdbc-test' is a separate module and it has it's own classpath.

Hmm, I wonder will init scripting work with testng and separate runner. Need to check.

Reply:
I think using classloader resources will be enough here.